modules/version/version.go

```diff
@@ -20,6 +22,10 @@ var (
 	Dirty = "{dirty}"
 )
 
+var (
+	re = regexp.MustCompile(`^Version:([^@]*)@Revision:([^\(]+)`)
```

Comment:
semver library doesn't handle case like that ?
So I guess this notation is not a valid semantic version ?

Reply:
This is the build version, set by the make file it simply tags the binary with a branch and a git revision number. It does not relate to the version of the release.

I use the git revision hash to check if both binaries are identical or not.